kfdefs/overlays/moc/smaug/opf-jupyterhub/alerts.yaml

```diff
@@ -26,3 +26,53 @@ spec:
             < 0.1
           labels:
             severity: critical
+    - name: SLOs-probe_success
+      rules:
+        - alert: RHODS Probe Success Burn Rate
+          annotations:
+            message: 'High error budget burn for {{ $labels.instance }} (current value: {{ $value }}).'
```

Comment:
I don't think we have a label `instance` in our metrics

Reply:
fixed it. I hope the fix is correct! I verified it is syntactically correct